### PR TITLE
fix flaky `bun-create.test.ts` with git templates

### PR DESCRIPTION
**Pull request by dylan-conway**
### What does this PR do?
checks for both `GITHUB_TOKEN` and `GITHUB_ACCESS_TOKEN` in `bun create` and `bun upgrade`

### How did you verify your code works?
CI

### CODE REVIEW

File: src/cli/upgrade_command.zig [hunk 1]
```diff
@@ -211,7 +211,7 @@ pub const UpgradeCommand = struct {
             ),
         );
 
-        if (env_loader.map.get("GITHUB_ACCESS_TOKEN")) |access_token| {
+        if (env_loader.map.get("GITHUB_TOKEN") orelse env_loader.map.get("GITHUB_ACCESS_TOKEN")) |access_token| {
             if (access_token.len > 0) {
                 headers_buf = try std.fmt.allocPrint(allocator, default_github_headers ++ "Access-TokenBearer {s}", .{access_token});
```
Comment (Electroid) [inline src/cli/upgrade_command.zig]: Where did `Access-TokenBearer` come from? Doesn't appear here: https://docs.github.com/en/rest/authentication/authenticating-to-the-rest-api?apiVersion=2022-11-28

Comment (dylan-conway) [reply src/cli/upgrade_command.zig]: updated

File: src/cli/upgrade_command.zig [hunk 2]
```diff
@@ -211,19 +211,19 @@ pub const UpgradeCommand = struct {
             ),
         );
 
-        if (env_loader.map.get("GITHUB_ACCESS_TOKEN")) |access_token| {
+        if (env_loader.map.get("GITHUB_TOKEN") orelse env_loader.map.get("GITHUB_ACCESS_TOKEN")) |access_token| {
             if (access_token.len > 0) {
-                headers_buf = try std.fmt.allocPrint(allocator, default_github_headers ++ "Access-TokenBearer {s}", .{access_token});
+                headers_buf = try std.fmt.allocPrint(allocator, default_github_headers ++ "AuthorizationBearer {s}", .{access_token});
```
Comment (Electroid) [inline src/cli/upgrade_command.zig:216]: I think this should just be `Bearer`?

Comment (Electroid) [reply src/cli/upgrade_command.zig:216]: `Authorization` is the header name.
`Bearer {s}` is the header value.

Comment (dylan-conway) [reply src/cli/upgrade_command.zig:216]: The names and values are in the same buffer, this is just allocating enough space for both. The entry stores the offsets and lengths for `Authorization` and `Bearer <token>`

**Pull request merged**